src/open_inwoner/scss/components/Contacts/Contacts.scss

```diff
@@ -0,0 +1,53 @@
+.approval {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--color-accent-lighter);
+  margin-top: var(--gutter-width);
+  border-radius: var(--border-radius);
+
+  &__header {
+    padding: var(--spacing-medium);
```

Comment:
You include this padding into all child tags of `.approval`. Whi not to change the padding of the parent class itself?

Reply:
Discussed, as long as this is consistent with the design this is fine